png_cicp_chunk_proposal.md

```diff
@@ -13,31 +13,33 @@ An existing W3C group note, [BT2100-in-PNG]  specifies an approach which is limi
 * extensible signaling of color space based on H.273
 * does not require the presence of iCCP chunk and embedded ICC profiles
 
+## Non Requirements
+* compatibility with HDR colorspace definitions in any other raster image format (eg. JPEG, AVIF)
+
 ## Strawman approach
-[H.273](https://www.itu.int/rec/T-REC-H.273/en) specifies a controlled vocabulary for the parameterization of
-color space information.
+[H.273](https://www.itu.int/rec/T-REC-H.273/en) specifies a controlled vocabulary for the parameterization of color space information.
 
-Define a cICP chunk that contains the 7 bytes necessary to carry the
-H.273 color space parameters:
+Define a `cICP` chunk that contains the 7 bytes necessary to carry the H.273 color space parameters:
 
-* COLPRIMS, 2 bytes, One of the ColourPrimaries enumerated values specified in Rec. ITU-T H.273 | [ISO/IEC 23091-2]
-* TRANSFC, 2 bytes, One of the TransferCharacteristics enumerated values specified in Rec. ITU-T H.273 | [ISO/IEC 23091-2]
-* VIDFRNG, 1 byte, Value of the VideoFullRangeFlag specified in Rec. ITU-T H.273 | [ISO/IEC 23091-2]
+* **COLPRIMS**, 2 bytes, One of the ColourPrimaries enumerated values specified in Rec. ITU-T H.273 | [ISO/IEC 23091-2]
+* **TRANSFC**, 2 bytes, One of the TransferCharacteristics enumerated values specified in Rec. ITU-T H.273 | [ISO/IEC 23091-2]
+* **VIDFRNG**, 1 byte, Value of the VideoFullRangeFlag specified in Rec. ITU-T H.273 | [ISO/IEC 23091-2]
 
-[ed.: these are inspired from recent JPEG standards that incorporate
-H.273 color space parameters]
+[ed.: these are inspired from recent JPEG standards (eg. JPEG-XL) that incorporate H.273 color space parameters]
 
 [ed.: The MATOEFFS parameter is not included because PNG does not support YCbCr]
```

Comment:
We need to put this back in to support YCbCr

Reply:
Revising PNG to support YCbCr would be a major undertaking.

Reply:
> Revising PNG to support YCbCr would be a major undertaking.

@svgeesus Can you explain why this is the case if sampling is 4:4:4?

Reply:
The [PNG color types](https://www.w3.org/TR/PNG/#6Colour-values) are


PNG image type | Colour type
-- | --
Greyscale | 0
Truecolour | 2
Indexed-colour | 3
Greyscale with alpha | 4
Truecolour with alpha | 6

There is no option in PNG to have YUV. It could be added, but no PNG software would support it and, since it is the image data, it can't be ignored. This would be a major, breaking, revision to a mature and widely deployed specification.

Reply:
@ProgramMax @lrosenthol I suggest that this PR exclude YUV. We can then file a new issue to cover it.

Reply:
I have created issue #16 to track/discuss and pull request #17 to implement.